apps/examples/src/styles.css

```diff
@@ -88,12 +86,65 @@ html,
 	}
 }
 
+.accordion__trigger{
+	border: none;
+	background: none;
+	cursor: pointer;
+	width: 100%;
+	text-align: left;
+	font-size: 10px;
+}
+
+.accordion__trigger__chevron{
+	color:rgba(0, 0, 0, 0.3);
+	transition: transform 300ms;
+	-webkit-transition: transform 300ms;
+	-moz-transition: transform 300ms;
+	-ms-transition: transform 300ms;
+	-o-transition: transform 300ms;
```

Comment:
you don't need to do the browser prefixes anymore
(i'm assuming that we're using postcss or something as the processor here, i could be wrong) - but you just need to do `transition`